tests/monsterdata_test.json

```diff
@@ -1,78 +1,78 @@
 {
-  pos: {
```

Comment:
I guess there's no flags to this JSON parser to allow unquoted keys? this is a common "extension".

Reply:
It could parse unquoted keys. but `values` couldn't.

```js
// NG
test2: Green,

// OK
test2: "Green",
```

May I quote the string values of this file?

Reply:
Yes, only for fields.

Reply:
Thank you very much!